docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/utils/index.md

```diff
@@ -221,26 +225,36 @@ If you have more than one session or user identifier, you can specify multiple e
 {`
 [
   {"schema": "my_custom_context", "field": "internal_user_id"},
-  {"table": "atomic", "field": "domain_userid"}
+  {"schema": "atomic", "field": "domain_userid"}
 ]
 `}
 </code></pre>
 </TabItem>
-<TabItem value="r&s" label="Redshift & Postgres" default>
+<TabItem value="redshift+postgres" label="Redshift & Postgres" default>
 <pre><code className="language-json">
 {`
 [
-  {"table": "my_custom_context", "field": "internal_user_id", "prefix": "mcc", "alias": "mcc_iud"},
-  {"table": "atomic", "field": "domain_userid"}
+  {"schema": "my_custom_context", "field": "internal_user_id", "prefix": "mcc", "alias": "mcc_iud"},
+  {"schema": "atomic", "field": "domain_userid"}
 ]
 `}
 </code></pre>
+
+For Redshift & Postgres we also introduce the `prefix` and `alias` fields, where `prefix` is the `prefix` that is put infront of each field name in the context, and `alias` is the table alias used upon joining. This can be useful when you are using custom SQL. As an example, using the above configuration we could access the `internal_user_id` field using the following SQL:
+
+```sql
+mcc_iud.mcc_internal_user_id as internal_user_id,
+```
+
+This could be leveraged in the `snowplow__custom_sql` variable. For more examples, please see [the following page](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-utils-advanced-operation).
```

Comment:
This isn't detailed in the configuration page? Also, we don't seem to talk about the `session_sql` anywhere I can find, and I can't even really remember how it gets used for postgres/redshift?